packages/ses/src-xs/index.js

```diff
@@ -0,0 +1,463 @@
+// @ts-nocheck
+/// <refs types="../types.js"/>
+
+import '../src/assert-shim.js';
+import '../src/console-shim.js';
+import { repairIntrinsics } from '../src/lockdown.js';
+
+import {
+  makeCompartmentConstructor as makeShimCompartmentConstructor,
+  compartmentOptions,
+} from '../src/compartment.js';
```

Comment:
Minor:

This file is intended to be an xs-flavored alternative to ses/index.js. Two minor changes would help a reader see and understand the parallelism:
- Move ses/index.js to ses/src/index.js, leaving behind a forwarding stub if appropriate.
- Rearrange the import order in that file to
   ```js
    import './assert-shim.js';
    import './console-shim.js';
    import './lockdown-shim.js';
    import './compartment-shim.js';
   ```
   so the reader doesn't need to worry about whether the difference in import order is significant.

Reply:
Might it even make sense to break this file up into a `ses/src-xs/lockdown-shim.js` and `ses/src-xs/compartment-shim.js`, and then change `ses/src-xs/index.js` to be that much more similar?

Reply:
I’ve taken this advice. You will find more files in `src-xs` on your next review.